Sources/SpotHeroEmailValidator/SHValidationResult.swift

```diff
@@ -1,22 +1,12 @@
-//  Copyright © 2019 SpotHero, Inc. All rights reserved.
-//
-//  Licensed under the Apache License, Version 2.0 (the "License");
-//  you may not use this file except in compliance with the License.
-//  You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-//  Unless required by applicable law or agreed to in writing, software
-//  distributed under the License is distributed on an "AS IS" BASIS,
-//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-//  See the License for the specific language governing permissions and
-//  limitations under the License.
-//
+// Copyright © 2020 SpotHero, Inc. All rights reserved.
 
 import Foundation
 
 public class SHValidationResult: NSObject {
+    /// Indicates whether or not the email address being analyzed is in valid syntax and format or not.
```

Comment:
Removing double "not"
```suggestion
    /// Indicates whether or not the email address being analyzed is in valid syntax and format.
```